website/docs/docs/dbt-versions/release-notes.md

```diff
@@ -20,6 +20,7 @@ Release notes are grouped by month for both multi-tenant and virtual private clo
 
 ## December 2024
 
+- **Fix**: [The dbt Semantic Layer](/docs/use-dbt-semantic-layer/dbt-sl) now respects the `execution_project` setting, including for exports.
```

Comment:
Looks good! I wonder if we should mention this is specifically for BigQuery?

Reply:
good shout thank you @aiguofer ! updated it now to this:

- **Fix**: [The dbt Semantic Layer](/docs/use-dbt-semantic-layer/dbt-sl) now respects the BigQuery [`execution_project` attribute](/docs/core/connect-data-platform/bigquery-setup#execution-project), including for exports.